frontend/src/authentication/contexts/CognitoAuthContext.js

```diff
@@ -23,147 +23,13 @@ Auth.configure({
   }
 });
 
-const initialState = {
-  isAuthenticated: false,
-  isInitialized: false,
-  user: null,
-  reAuthStatus: false,
-  requestInfo: null
-};
-
-const handlers = {
-  INITIALIZE: (state, action) => {
-    const { isAuthenticated, user } = action.payload;
-
-    return {
-      ...state,
-      isAuthenticated,
-      isInitialized: true,
-      user,
-      reAuthStatus: false
-    };
-  },
-  LOGIN: (state, action) => {
-    const { user } = action.payload;
-
-    return {
-      ...state,
-      isAuthenticated: true,
-      user
-    };
-  },
-  LOGOUT: (state) => ({
-    ...state,
-    isAuthenticated: false,
-    user: null
-  }),
-  REAUTH: (state, action) => {
-    const { reAuthStatus, requestInfo } = action.payload;
-
-    return {
-      ...state,
-      reAuthStatus,
-      requestInfo
-    };
-  }
-};
-
-const reducer = (state, action) =>
-  handlers[action.type] ? handlers[action.type](state, action) : state;
-
-export const CognitoAuthContext = createContext({
-  ...initialState,
-  platform: 'Amplify',
-  login: () => Promise.resolve(),
-  logout: () => Promise.resolve(),
-  reauth: () => Promise.resolve()
-});
-
 export const CognitoAuthProvider = (props) => {
   const { children } = props;
-  const [state, dispatch] = useReducer(reducer, initialState);
-
-  useEffect(() => {
-    const initialize = async () => {
-      try {
-        const user = await Auth.currentAuthenticatedUser();
-        dispatch({
-          type: 'INITIALIZE',
-          payload: {
-            isAuthenticated: true,
-            user: {
-              id: user.attributes.email,
-              email: user.attributes.email,
-              name: user.attributes.email
-            }
-          }
-        });
-      } catch (error) {
-        dispatch({
-          type: 'INITIALIZE',
-          payload: {
-            isAuthenticated: false,
-            user: null
-          }
-        });
-      }
-    };
-
-    initialize().catch((e) => dispatch({ type: SET_ERROR, error: e.message }));
-  }, []);
-
-  const login = async () => {
-    Auth.federatedSignIn()
-      .then((user) => {
-        dispatch({
-          type: 'LOGIN',
-          payload: {
-            user: {
-              id: user.attributes.email,
-              email: user.attributes.email,
-              name: user.attributes.email
```

Comment:
This info is now returned in GenericAuthContext right? But instead of returning `id` `email` and `name` we return the following in `frontend/src/authentication/contexts/GenericAuthContext.js:188`
```
        email: user.attributes.email,
        id_token: user.signInUserSession.idToken.jwtToken,
        short_id: 'none'
```

Does this influence any place where we use user.id or user.name? For example in `frontend/src/design/components/popovers/AccountPopover.js:64` we use `user.name`

Reply:
@noah-paige , you are right! . Yes we still have the user.name, user.email, user.id , and all three contains the `email` field for Cognito stack and also for the External Idp stack. 

Due to this this just works as expected with the `AccountPopover` and also whereever it is used elsewhere